subsys/bootloader/image/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+if (CONFIG_IS_S1_IMAGE)
+  return()
+endif()
+
```

Comment:
Need a comment explaining when subsys/bootloader/image/CMakeLists.txt is executed and why we don't need to run any of the below scripts when CONFIG_IS_S1_IMAGE.